components/edit-collective/sections/Host.js

```diff
@@ -94,9 +94,10 @@ class Host extends React.Component {
       <MessageBox type="info" fontSize="13px" withIcon>
         <FormattedMessage
           id="collective.edit.host.legalName.info"
-          defaultMessage="Please set the legal name of the host in the Info section under <SettingsLink>your settings</SettingsLink>. This is required if your legal name is different than your display name for tax and accounting purposes."
+          defaultMessage="Please set the legal name {isSelfHosted, select, false {of the host} other {}} in the Info section under <SettingsLink>your settings</SettingsLink>. This is required if your legal name is different than your display name for tax and accounting purposes."
```

Comment:
I would replace "your legal name" by "the legal name". And I would replace "your display name" by "the display name".

Reply:
"under your settings" -> "of the settings".

Reply:
Yes. Also, there's a double space here, isn't it?

Reply:
> I would replace "your legal name" by "the legal name". And I would replace "your display name" by "the display name".

Done. 👍🏽 

> "under your settings" -> "of the settings".

Done. 👍🏽 

> Yes. Also, there's a double space here, isn't it?

@Betree : I checked and it seems that `FormattedMessage` is smart enough to not display double spaces; so we don't need to deal with the spacing unless we explicitly need double spaces.